<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenByMenuBar"/>
    <addaction name="actionExitByMenuBar"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFolderByToolBar"/>
   <addaction name="actionExitByToolBar"/>
  </widget>
  <action name="actionOpenByMenuBar">
   <property name="icon">
    <iconset>
     <normalon>:/toolButton_images/asset images/folder_open_48px.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExitByMenuBar">
   <property name="icon">
    <iconset>
     <normalon>:/toolButton_images/asset images/exit_48px.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpenFolderByToolBar">
   <property name="icon">
    <iconset resource="asset.qrc">
     <normaloff>:/toolButton_images/asset images/folder_open_48px.png</normaloff>:/toolButton_images/asset images/folder_open_48px.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExitByToolBar">
   <property name="icon">
    <iconset>
     <normalon>:/toolButton_images/asset images/exit_48px.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="asset.qrc"/>
 </resources>
 <connections/>
</ui>
